cli/pkg/backend/aws/ecs_compute_backend.go

```diff
@@ -465,3 +466,69 @@ func (b *ECSComputeBackend) getTaskID(taskARN string) (string, error) {
 	}
 	return segments[len(segments)-1], nil
 }
+
+func (b *ECSComputeBackend) GetEvents(ctx context.Context, stackName string, services []string) error {
+	if len(services) == 0 {
+		return nil
+	}
+
+	clusterArn := b.Backend.Conf().GetClusterArn()
+
+	ecsClient := b.Backend.GetECSClient()
+
+	ecsServices := make([]string, 0)
+	for _, service := range services {
+		ecsService := fmt.Sprintf("%s-%s", stackName, service)
+		ecsServices = append(ecsServices, ecsService)
+	}
```

Comment:
I think we should avoid this convention throughout the codebase and move it into a function that is easier to change in the future. That is kind of what I did here: https://github.com/chanzuckerberg/happy/blob/main/cli/pkg/backend/aws/ecs.go#L20

Reply:
This is a copy of previously existing code; I can wrap it in a function, but it is a bit out of scope for this pull request.